### DISCUSSION TITLE
Commit 작성 규칙 (제안)

### DISCUSSION THREAD

**Discussion by Hyoj-Kim** [GunGonGamLeeLee/ft_transcendence#6] [0. 공지사항]
### 0️⃣ 기본 형식
[Conventional Commit](https://www.conventionalcommits.org/ko/v1.0.0-beta.4/)을 기준으로 작성해주세요.
> vscode 확장프로그램 Conventional Commits이나 [commitlint](https://commitlint.js.org/#/)를 사용하는 것을 추천드립니다.
<img width="300" alt="image" src="https://user-images.githubusercontent.com/76509884/193395420-506ae603-b361-44c2-9361-5ce7007c1d9d.png">

```
<타입>[적용 범위(선택 사항)]: <설명>

[본문(선택 사항)]

[꼬리말(선택 사항)]
```
```
feat: 무언가 새로운 기능 추가

어떤 기능 추가

- #이슈번호
```

### 1️⃣ 타입
```
feat
fix
docs
style
refactor
test
build
ci
chore
wip: 중간 점검을 위한 커밋 -> PR 전에 squash 하는 용도!
```

### 2️⃣ 제목, 본문
- 이해하기 쉽도록 한글로 작성해주세요.

### 3️⃣ 꼬리말
- 연관된 issue 번호를 `#`와 함께 기입해주세요. 해당 커밋이 push 되면 작성한 issue에 자동으로 연결됩니다.

**Comment by mypark42** [comment]
타입 중에 '아직 작업중'을 표시할 수 있는 방식이 있었으면 좋겠습니다

**Reply by Hyoj-Kim** [reply]
딱히 생각나는 키워드가 없어 wip로 하는게 어떨까 싶네요..! 어짜피 squash 할 거니까 합쳐진 커밋만 타입 제대로 골라주면 되겠죠...?ㅎㅎ
